src/HW1/task1/Program.java

```diff
@@ -0,0 +1,28 @@
+package HW1.task1;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * 1) реалізувати за допомогою concurrence пожежну частину, тобто є потоки які очікують сигналу,
```

Comment:
If you want to add a description of the task, then I would recommend adding README.md and writing everything in English